modules/github/github.go

```diff
@@ -0,0 +1,326 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor: Miles Crabill <mcrabill@mozilla.com>
+
+package github
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/google/go-github/github"
+	"go.mozilla.org/userplex/modules"
+	"golang.org/x/oauth2"
+)
+
+func init() {
+	modules.Register("github", new(module))
+}
+
+type module struct{}
+
+func (m *module) NewRun(c modules.Configuration) modules.Runner {
+	r := new(run)
+	r.Conf = c
+	return r
+}
+
+type run struct {
+	Conf         modules.Configuration
+	p            parameters
+	c            credentials
+	ghclient     *github.Client
+	githubToLdap map[string]string
+	ldapToGithub map[string]string
+}
+
+type organization struct {
+	Name  string
+	Teams []string
+}
+
+type parameters struct {
+	Organizations    []organization
```

Comment:
Is it wise to support several organizations at the same time, or should we limit one userplex module conf to one organization? My gut feeling is the later would cover the use case and avoid configuration mistakes.